### PR TITLE
chore: update deps to match "deps of deps" versions

### PR DESCRIPTION
**Pull request by dgozman**
This way we do not bring two copies of pngs and source-map.

### CODE REVIEW

File: test/base-reporter.spec.ts [hunk 1]
```diff
@@ -43,7 +43,7 @@ test('print the error name', async ({ runInlineTest }) => {
   });
   expect(result.exitCode).toBe(1);
   expect(result.failed).toBe(1);
-  expect(result.output).toContain('FooBarError: my-message');
```
Comment (JoelEinbinder) [inline test/base-reporter.spec.ts:46]: why does this change?

Comment (dgozman) [reply test/base-reporter.spec.ts:46]: No idea!

File: package.json [hunk 2]
```diff
@@ -43,9 +43,9 @@
     "ms": "^2.1.2",
     "pirates": "^4.0.1",
     "pixelmatch": "^5.2.1",
-    "pngjs": "^5.0.0",
```
Comment (JoelEinbinder) [inline package.json:46]: what package are we matching with here?

File: test/base-reporter.spec.ts (hunk 1 above)
Comment (JoelEinbinder) [reply test/base-reporter.spec.ts:46]: Seems broken!

File: test/base-reporter.spec.ts [hunk 3]
```diff
@@ -59,7 +59,7 @@ test('print should print the error name without a message', async ({ runInlineTe
   });
   expect(result.exitCode).toBe(1);
   expect(result.failed).toBe(1);
-  expect(result.output).toContain('FooBarError: ');
+  expect(result.output).toContain('FooBarError');
```
Comment (JoelEinbinder) [inline test/base-reporter.spec.ts:62]: why this change?

**Review by JoelEinbinder — APPROVED** [review]
